batch_seqr_loader/batch_workflow.py

```diff
@@ -1276,54 +1227,61 @@ def _make_joint_genotype_jobs(
                     number_of_samples=len(sample_names_will_be_in_db),
                     interval_idx=idx,
                     number_of_intervals=utils.NUMBER_OF_GENOMICS_DB_INTERVALS,
-                    interval=intervals_j.intervals[f'interval_{idx}'],
+                    interval=intervals[f'interval_{idx}'],
+                    output_vcf_path=joint_called_vcf_path,
                 )
                 if import_gvcfs_job_per_interval.get(idx):
                     genotype_vcf_job.depends_on(import_gvcfs_job_per_interval.get(idx))
 
-                if not is_small_callset:
+                if len(samples) > 1000:
```

Comment:
Was this intentional? I thought you had `is_small_callset` etc. to avoid magic thresholds in the code.

Reply:
Was in the process of refactoring that better in the production-pipelines. Put `is_small_callset` back now!